src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -146,22 +83,15 @@ public AgentSettings LoadSettings()
                     WriteSection(StringUtil.Loc("EulasSectionHeader"));
 
                     // Verify the EULA exists on disk in the expected location.
-                     string eulaFile = Path.Combine(IOUtil.GetExternalsPath(), Constants.Path.TeeDirectory, "license.html");
+                    string eulaFile = Path.Combine(IOUtil.GetExternalsPath(), Constants.Path.TeeDirectory, "license.html");
                     IOUtil.AssertFile(eulaFile);
 
                     // Write elaborate verbiage about the TEE EULA.
                     _term.WriteLine(StringUtil.Loc("TeeEula", eulaFile));
                     _term.WriteLine();
 
                     // Prompt to acccept the TEE EULA.
-                    acceptTeeEula =
-                        (flags?.Contains(CliArgs.AcceptTeeEula) ?? false) ||
-                        promptManager.ReadBool(
-                            argName: CliArgs.AcceptTeeEula,
-                            description: StringUtil.Loc("AcceptTeeEula"),
-                            defaultValue: false,
-                            args: null,
-                            unattended: unattended);
+                    acceptTeeEula = command.GetAcceptTeeEula();
```

Comment:
The addition of the CommandSettings class eliminates tedious concern from the configuration manager classes. This allows the configuration manager classes to focus on what they need to do: configure. The tedium of checking supplied flags and prompting details has been moved into the CommandSettings class.